view/graph/Graph.kt

```diff
@@ -258,7 +258,7 @@ class Graph(private val interactions: Interactions) {
         }
     }
 
-    class Reasoner {
```

Comment:
"Reasoner" sounded too much like we actually had a Reasoner embedded in the graph. What it actually is is reasoning data.